standards/.pylintrc

```diff
@@ -135,7 +135,9 @@ enable-variables=yes
 # Tells wether we should check for unused import in __init__ files.
 init-import=no
 
-
+# A regular expression matching the name of dummy variables (i.e. expectedly
+# not used).
+dummy-variables-rgx=_$|dummy
```

Comment:
I thought as a compromise we could allow dummy*. Does that work with the current or not?

Sent from a mobile device. Please excuse my brevity or transcription errors.

On Oct 7, 2015, at 14:27, ticoann <notifications@github.com<mailto:notifications@github.com>> wrote:


In standards/.pylintrc<https://github.com/dmwm/WMCore/pull/6302#discussion_r41434557>:

> @@ -135,7 +135,9 @@ enable-variables=yes
>  # Tells wether we should check for unused import in __init__ files.
>  init-import=no
>
> -
> +# A regular expression matching the name of dummy variables (i.e. expectedly
> +# not used).
> +dummy-variables-rgx=_$|dummy


are we allowing only _ and dummy for dummy variable or any thing start with dummy prefix?

—
Reply to this email directly or view it on GitHub<https://github.com/dmwm/WMCore/pull/6302/files#r41434557>.